dev/workflow/applications/sfs.py

```diff
@@ -164,7 +166,12 @@ def get_task_names(self):
         if options['do_extractvars']:
             tasks += ['extractvars']
 
-        # TODO: Add archive
+        #TODO add archiving for SFS
+        #if options['do_archcom']:
+        #    tasks += ['arch_tars']
+        #    if options['do_globusarch']:
+        #        tasks += ['globus']
+
```

Comment:
To appease the linter:
```suggestion
        # TODO add archiving for SFS
        # if options['do_archcom']:
        #     tasks += ['arch_tars']
        #     if options['do_globusarch']:
        #         tasks += ['globus']

```